docs/api/csf/index.mdx

```diff
@@ -35,6 +41,16 @@ For more examples, see [writing stories](../../writing-stories/index.mdx).
 
 With CSF, every named export in the file represents a story object by default.
 
+<If renderer="svelte">
+
+<Callout variant="info">
+
+With Svelte CSF, you use the Story component instead of a named export to define a story. The name of the story, shown in the sidebar, is defined by the `name` attribute of the Story component. Annotations, such as `args`, `decorators`, and `parameters`, are defined as attributes on the Story component.
```

Comment:
```suggestion
With Svelte CSF, you use the `Story` component returned by `defineMeta` instead of a named export to define a story. The name of the story, shown in the sidebar, is defined by the `name` attribute of the Story component. Annotations, such as `args`, `decorators`, and `parameters`, are defined as attributes on the Story component.
```